x-pack/plugins/index_lifecycle_management/__jest__/client_integration/app/app.helpers.tsx

```diff
@@ -17,7 +18,13 @@ const breadcrumbService = createBreadcrumbsMock();
 
 const AppWithContext = (props: any) => {
   return (
-    <KibanaContextProvider services={{ breadcrumbService, license: licensingMock.createLicense() }}>
+    <KibanaContextProvider
+      services={{
+        breadcrumbService,
+        license: licensingMock.createLicense(),
+        managementPageLayout: KibanaPageTemplate,
```

Comment:
**Step 4:** Update Jest tests to set up testbed by stubbing the injected `managementPageLayout` with `KibanaPageTemplate`.